src/index.js

```diff
@@ -522,7 +510,7 @@ export async function getPersonalizationForView(viewName = '__view__') {
  *                                (retrieve them using `getPersonalizationForView`)
  * @returns a promise that the propositions were applied
  */
-export async function applyPersonalization(viewName, propositions) {
+export async function applyPersonalization(viewName, propositions, forcibly = false) {
```

Comment:
Let the user decide if propositions should be reapplied forcibly for the view, or if we only do it once and then cache the results